runtime/port/unix/j9sysinfo.c

```diff
@@ -1675,8 +1675,11 @@ j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, const J9CacheInfoQue
 	) {
 		/* L1 data cache line size */
 		int32_t rc = (int32_t)sysconf(_SC_LEVEL1_DCACHE_LINESIZE);
-		if (rc >= 0) {
+		if (rc > 0) {
 			result = rc;
+		} else if (rc == 0) {
+			// Cache line size in unavailable on some systems
+			result = 64;
```

Comment:
I had suggested 64 in the issue where this was discussed but have no experience on aarch64 to tell if that's actually reasonable.  Does 64 make sense given the chips that are available?  Would another value be a safer default?

Reply:
As I understand it, it's better if the number is too big than too small. If it's too big then a little memory will be wasted, but if it's too small then it will defeat the purpose of the optimization and affect performance.

Reply:
The line size of L1 data cache of Cortex-A53 (low-end AArch64 core) is 64 bytes, as stated in http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0500j/BABCFDAH.html.
Other Cortex-A3x, A5x and A7x chips also seem to have 64 byte cache line in L1 data cache.

Reply:
I don't think there is a good reason to choose other line size.  Cortex AArch64 cores seem to have the 64 byte cache lins.

Reply:
Pls add the information https://github.com/eclipse/openj9/pull/9155/files#r404837464 as a comment in the code for future reference.

Reply:
Updated the comment.